rosbridge_server/scripts/rosbridge_websocket.py

```diff
@@ -125,6 +139,48 @@ def shutdown_hook():
             print "--max_message_size argument provided without a value. (can be None or <Integer>)"
             sys.exit(-1)
 
+    if "--topics_glob" in sys.argv:
+        idx = sys.argv.index("--topics_glob") + 1
+        if idx < len(sys.argv):
+            value = sys.argv[idx]
+            if value == "None":
+                RosbridgeWebSocket.topics_glob = None
```

Comment:
Looks like RosbridgeWebSocket.topics_glob and RosbridgeWebSocket.services_glob aren't used anywhere else, except below, to initialize the Capabilities - is there a reason you set it?